Diagnostics/HealthChecker/Analyzer/Get-FilteredSettingOverrideInformation.ps1

```diff
@@ -0,0 +1,118 @@
+
+        if ($null -ne $filteredResults) {
+            Write-Verbose "Found $($filteredResults.Count) override(s)"
+            foreach ($entry in $filteredResults) {
+                Write-Verbose "Working on entry: $($entry.Name)"
+                foreach ($p in [array]($entry.Parameters)) {
+                    Write-Verbose "Working on parameter: $p"
+                    if ($p.Contains($FilterParameterName)) {
+                        $value = $p.Substring($FilterParameterName.Length + 1) # Add plus 1 for '='
+                        # everything matched, however, only add it to the list for the following reasons
+                        # - Status is Accepted and not from AD and a unique value in the list
+                        # - Or From AD and current logic determines it applies
+
+                        if ($usedAdSettings) {
+                            # can have it apply by build and server parameter
+                            if (($null -eq $entry.MinVersion -or
+                                    $FilterServerVersion -ge $entry.MinVersion) -and
+                                (($null -eq $entry.MaxVersion -or
+                                    $FilterServerVersion -le $entry.MaxVersion)) -and
+                                (($null -eq $entry.Server -or
+                                    $entry.Server.ToLower().Contains($adjustedFilterServer)))) {
+                                $status = $entry.Status
+                            } else {
+                                $status = "DoesNotApply"
+                            }
+                        } else {
+                            $status = $entry.Status
+                        }
+
+                        if ($status -eq "Accepted" -and
+                            ($results.Count -lt 1 -or
+                            -not ($results.ParameterValue.ToLower().Contains($value.ToLower())))) {
+                            $results.Add([PSCustomObject]@{
+                                    Name           = $entry.Name
+                                    Reason         = $entry.Reason
+                                    ModifiedBy     = $entry.ModifiedBy
+                                    ComponentName  = $entry.ComponentName
+                                    SectionName    = $entry.SectionName
+                                    ParameterName  = $FilterParameterName
+                                    ParameterValue = $value
+                                    Status         = $entry.Status
+                                    TrueStatus     = $status
+                                    FromAdSettings = $usedAdSettings
+                                })
+                        } elseif ($status -eq "Accepted") {
+                            Write-Verbose "Already have 1 Accepted value added to list no need to add another one. Skip adding $($entry.Name)"
```

Comment:
This would mean that we filter out all duplicate overrides which enable a feature (e.g., SerializedDataSining) with the status `Accepted` and so, we don't show them in the final output. If that's what we wanted to do, it works as expected. Otherwise, we might need to adjust the logic to not exclude duplicate items.

<img width="688" alt="image" src="https://user-images.githubusercontent.com/40993616/224097728-85e6bcb2-b2b9-4cf4-8938-4e2234e52699.png">

<img width="635" alt="image" src="https://user-images.githubusercontent.com/40993616/224098163-bd2bb32a-d452-44f2-bf67-bd769f198085.png">